src/strings/normalize.c

```diff
@@ -578,11 +632,12 @@ static void grapheme_composition(MVMThreadContext *tc, MVMNormalizer *n, MVMint3
  * compute the normalization. */
 MVMint32 MVM_unicode_normalizer_process_codepoint_full(MVMThreadContext *tc, MVMNormalizer *n, MVMCodepoint in, MVMCodepoint *out) {
     MVMint64 qc_in, ccc_in;
-
+    int is_prepend = is_grapheme_prepend(tc, in);
     /* If it's a control character (outside of the range we checked in the
      * fast path) then it's a normalization terminator. */
-    if (in > 0xFF && is_control_beyond_latin1(tc, in))
+    if ( ( in > 0xFF && is_control_beyond_latin1(tc, in) ) && ( !is_prepend ) ) {
```

Comment:
What are the cases when `is_control_beyond_latin1` can return something that is a prepend, out of curiosity?